frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -655,6 +679,23 @@ export const surveyLogic = kea<surveyLogicType>([
                 5
             )
         },
+        setPropertyFilters: () => {
+            // Reload all survey data when filters change
+            if (values.survey.questions) {
+                values.survey.questions.forEach((question, index) => {
+                    if (question.type === SurveyQuestionType.Rating) {
```

Comment:
nit: I think a `switch` here would make the code more readable.

Reply:
i agree, will fix that on https://github.com/PostHog/posthog/pull/28466, which is the PR on top of this, as this logic already changed a little bit